index.js

```diff
@@ -61,3 +61,4 @@ export { default as NoteCreatePage } from './lib/Notes/NoteCreatePage';
 export { default as NoteViewPage } from './lib/Notes/NoteViewPage';
 export { default as NoteEditPage } from './lib/Notes/NoteEditPage';
 export { default as NotesSmartAccordion } from './lib/Notes/NotesSmartAccordion';
+export * from './lib/CustomFields';
```

Comment:
It looks like you're just exporting one thing from lib/CustomFields. Can you avoid exporting `*` and just write out the export fully so it's easier for devs using this to figure out names of exports and what's available? Ie, it's a lot easier to just look at the top level index than having to go into subdirectories.

Reply:
Good point! Made a change